example/operator/20-garden.yaml

```
@@ -42,6 +42,11 @@ spec:
         enabled: false
   # volume:
   #   minimumSize: 20Gi
+  # certManagement:
+  #   defaultIssuer:
+  #     email: some.name@my-email-domain.com
+  #     #secretRef: defaultIssuerPrivateKey
```

```suggestion
  #      secretRef: defaultIssuerPrivateKey # optionally use a pre-defined ACME account
```

Optional fields are only commented "once" in this repo. We should add a comment, if it needs more explanation, otherwise it looks like a mistake.